packages/volto-razzle/README.md

```diff
@@ -0,0 +1,69 @@
+![repo-banner](https://user-images.githubusercontent.com/4060187/28923990-050a32d4-782e-11e7-9da7-574ce5a8b455.png)
```

Comment:
I would add a disclaimer at the top, specifying it's a fork and the reason behind. Maintaining attibutions.

Reply:
Done.